best-practices/rspec.md

```diff
@@ -0,0 +1,30 @@
+## Rspec style guide
+
+- First #describe what you are doing
+- Then establish the #context
+- #it only expects one thing (within reason)
+- Prefer explicitness - #it #its #specify #subject are nice shortcuts but
+consider whether they are sacrificing readability
```

Comment:
I've read the recommendation:
````
describe '#method_name' do
  context 'with students in teachers class' do # setup
    it 'does the thing' do
```
and I kinda like it. That's only for unit tests of course.